default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  # crane flake
+  crane,
+  toolchainOverride ? null,
+  # standard
+  pkgs,
+  lib,
+  # dependencies
+  pkg-config,
+  llvm,
+  clang,
+  openssl,
+  makeWrapper,
```

Comment:
We prolly don't need these dependencies. Llvm and clang are required for bindgen, and openssl is required for, well, ssl. We have neither here.

We definitely don't need `makeWrapper`.